be/src/column/const_column.cpp

```diff
@@ -73,4 +73,19 @@ void ConstColumn::check_or_die() const {
     _data->check_or_die();
 }
 
+StatusOr<ColumnPtr> ConstColumn::upgrade_if_overflow() {
+    if (_size > Column::MAX_CAPACITY_LIMIT) {
+        return Status::InternalError("Size of ConstColumn exceed the limit");
+    }
+    auto ret = _data->upgrade_if_overflow();
```

Comment:
for a valid const column, data column size should be always 1. I think we could only add DCheck here.

Reply:
It may be a array column, Theoretically, it will also exceed the limit。

Reply:
It could also be a super long string

Reply:
This is a bit more rigorous